Parsers/ASimUserManagement/Parsers/vimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,541 @@
+           and (isnull(endtime) or TimeGenerated <= endtime)
+      | where Facility == "authpriv"
+      | where (array_length(actorusername_has_any) == 0 and 
+          array_length(targetusername_has_any) == 0 or (SyslogMessage has_any(targetusername_has_any))) and
+          (array_length(eventtype_in) == 0 or ("GroupDeleted" in (eventtype_in))) and
+          (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(HostIP, srcipaddr_has_any_prefix))
+      | where ProcessName == "groupdel"
+      | parse SyslogMessage with "group '" GroupName "' removed" *
+      | extend 
+          EventType   = "GroupDeleted",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Computer in (_ASIM_GetSourceBySourceType('LinuxAuthpriv'))
+      | where (isnull(starttime) or TimeGenerated >= starttime)
+           and (isnull(endtime) or TimeGenerated <= endtime)
+      | where Facility == "authpriv"
+      | where (array_length(targetusername_has_any) == 0 or (SyslogMessage has_any(targetusername_has_any))) and 
+          (array_length(actorusername_has_any) == 0 or (SyslogMessage has_any(actorusername_has_any))) and 
+          (array_length(eventtype_in) == 0 or ("UserAddedToGroup" in (eventtype_in)) or ("UserRemovedFromGroup" in (eventtype_in))) and
+          (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(HostIP, srcipaddr_has_any_prefix))
+      | where ProcessName == "gpasswd"
+      | parse SyslogMessage with "user " TargetUsername " " Action " by " ActorUsername " " * " group " GroupName
+      | where (array_length(targetusername_has_any) == 0 or (TargetUsername has_any(targetusername_has_any))) and
+          (array_length(actorusername_has_any) == 0 or (ActorUsername has_any(actorusername_has_any)))
+      | lookup ActionLookup on Action
+      | where (array_length(eventtype_in) == 0 or (EventType in (eventtype_in)))
+      | project-away Action
+      | extend 
+          EventResult = "Success"
+      | invoke ItemParser()
+  )
+  | where not(disabled)
+  | invoke _ASIM_ResolveDvcFQDN ("HostName")
+  | lookup SeverityLookup on SeverityLevel
+  | project-rename 
+      ActingAppName = ProcessName,
+      DvcId         = _ResourceId,
+      EventUid      = _ItemId
+  | extend
+      ActingAppId         = tostring(ProcessID),
+      ActingAppType       = "Process",
+      ActorUsernameType   = iif(isnotempty(ActorUsername), "Simple", ""),
+      DvcIdType           = iff (DvcId == "", "", "AzureResourceID"),
+      DvcIpAddr           = iif(HostIP == "Unknown IP","",HostIP),
+      DvcOs               = "Linux",
+      EventCount          = int(1),
+      EventEndTime        = TimeGenerated,
+      EventProduct        = "Authpriv",
+      EventSchema         = "UserManagement",
+      EventSchemaVersion  = "0.1.1",
+      EventStartTime      = TimeGenerated,
+      EventUid            = _ItemId,
```

Comment:
@KustoKing 
pls remove this, we are renaming at line 336

Reply:
Done